vms/platformvm/vm.go

```diff
@@ -55,6 +57,9 @@ import (
 )
 
 var (
+	// How often the mempool txs will be re-verified and pruned
+	mempoolPrunerFrequency = 6 * time.Hour
```

Comment:
I feel like this should be a lot lower

Reply:
From looking at mainnet metrics, the P-chain mempool seems to grow at a slow rate. What do you think would be a good number? I'm thinking every 30 minutes?

Reply:
30m seems reasonable for now